docs/source/en/preprocessing.mdx

```diff
@@ -14,37 +14,33 @@ specific language governing permissions and limitations under the License.
 
 [[open-in-colab]]
 
-Before you can use your data in a model, the data needs to be processed into an acceptable format for the model. A model does not understand raw text, images or audio. These inputs need to be converted into numbers and assembled into tensors. In this tutorial, you will:
+Before you can train a model on a dataset, it needs to be preprocessed into the expected model input format. Whether your data is text, images, or audio, they need to be converted and assembled into batches of tensors. 🤗 Transformers provides a set of preprocessing classes to help prepare your data for the model. In this tutorial, you'll learn how to use:
 
-* Preprocess textual data with a tokenizer.
-* Preprocess image or audio data with a feature extractor.
-* Preprocess data for a multimodal task with a processor.
+* A [Tokenizer](./main_classes/tokenizer) to convert text into a sequence of tokens, create a numerical representation of the tokens, and then tensors.
+* A [Feature extractor](./main_classes/feature_extractor) extracts sequential features from audio waveforms and images and converts them into tensors.
+* A [Processor](./main_classes/processor) combines a tokenizer and a feature extractor for multimodal inputs.
 
-## NLP
+## Tokenizer
```

Comment:
I liked having the modalities as header. Not sure class names will speak to the user as much.